decidim-meetings/app/helpers/decidim/meetings/application_helper.rb

```diff
@@ -16,7 +16,7 @@ module ApplicationHelper
       def filter_origin_values
         origin_values = []
         origin_values << TreePoint.new("official", t("decidim.meetings.meetings.filters.origin_values.official"))
-        origin_values << TreePoint.new("citizens", t("decidim.meetings.meetings.filters.origin_values.citizens")) # todo
+        origin_values << TreePoint.new("participants", t("decidim.meetings.meetings.filters.origin_values.participants")) # todo
```

Comment:
Maybe you need to change or remove the key for other languages as well?

Reply:
Closing per : https://github.com/decidim/decidim/pull/8697#discussion_r783009123